addons/pos_self_order/static/src/app/components/product_info_popup/product_info_popup.js

```diff
@@ -2,7 +2,7 @@ import { Component, useExternalListener, useState } from "@odoo/owl";
 
 export class ProductInfoPopup extends Component {
     static template = "pos_self_order.ProductInfoPopup";
-    static props = ["product", "addToCart"];
+    static props = ["product", "addToCart", "title", "close"];
```

Comment:
the close is used in the code, but the title is set there:

https://github.com/odoo/odoo/blob/e36319ae8821fc333590af6906ecf6c4b9b60f00/addons/pos_self_order/static/src/app/components/product_card/product_card.js#L119-L121

but I don't see it used in the JS class or XML, so maybe it should be removed from the call instead.

Reply:
makes sense, will update it.